product_template_tags_code/models/product_template_tag.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+from odoo.addons.http_routing.models.ir_http import slugify
+
+
+class ProductTemplateTag(models.Model):
+
+    _inherit = "product.template.tag"
+
+    code = fields.Char(string="Code")
```

Comment:
what happened to the `compute` version?

Reply:
the compute finally don't work with the @sebalix advice I try this solution

the compute was not use in the record creation

Reply:
but then why not using an inverse func?

Reply:
cause @sebalix tell me it's easier like this, but I can change :wink:

Reply:
weird, it works for me for both create and write :thinking:

Reply:
We didn't test the inverse actually (there was only the compute), and I was tired, and newbie, sooo... :p